go/vt/discovery/tablet_picker_test.go

```diff
@@ -169,7 +171,17 @@ func addTablet(te *pickerTestEnv, id int, tabletType topodatapb.TabletType, serv
 }
 
 func deleteTablet(te *pickerTestEnv, tablet *topodatapb.Tablet) {
-	te.topoServ.DeleteTablet(context.Background(), tablet.Alias)
+
+	//log error
+	if err2 := te.topoServ.DeleteTablet(context.Background(), tablet.Alias) ; err2 != nil {
+		log.Errorf("te.topoServ.DeleteTablet(context.Background(), tablet.Alias) failed : %v", err2)
```

Comment:
more readable error message like `"failed to DeleteTablet with alias %v: %v", tablet.Alias, err`